deploy/static/02-trivy-operator.rbac.yaml

```diff
@@ -242,3 +228,68 @@ subjects:
   - kind: ServiceAccount
     name: trivy-operator
     namespace: trivy-system
+---
+# Source: trivy-operator/templates/leader_election.yaml
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: trivy-operator-leader-election
+  namespace: trivy-system
+  labels:
+    app.kubernetes.io/name: trivy-operator
+    app.kubernetes.io/instance: trivy-operator
+    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/managed-by: kubectl
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
```

Comment:
when `leases` was in `ClusterRole` it use to have only verbs (create, get, update) now in `Role` has additional (list, watch, patch, delete).
advice on this change

Reply:
Allowing same verbs on the namespace level now.